ddtrace/opentelemetry/span.go

```diff
@@ -45,6 +48,35 @@ func (s *span) SetName(name string) {
 	s.attributes[ext.SpanName] = strings.ToLower(name)
 }
 
+// SpanEvent holds information about span events
+type SpanEvent struct {
```

Comment:
```suggestion
type SpanEvent struct {
```

Do we need to export this? Could it be an internal type?